### PR TITLE
"review_state" and "subjects" usable as facet

### PR DESCRIPTION
**Pull request by ksuess**
Add celery task "deleteindex"

### CODE REVIEW

File: examples/analysis/mappings-german-analysis.json [hunk 1]
```diff
@@ -1,399 +1,364 @@
...
+    },
+    "zope.schema._field.List": {
+        "type": "keyword"
+    },
+    "zope.schema._field.Tuple": {
+        "type": "keyword"
+    },
+    "zope.schema._field.URI": {
+        "type": "text"
+    }
```
[the remark this answers is not in the document]
Comment (ksuess) [reply examples/analysis/mappings-german-analysis.json:363]: Mappings is reorganized and stripped:
- Text, Textline, subjects with analyzer at the top of the file. Everything else below.
- No more special "title" configuration, as it is Text

File: examples/analysis/elasticsearch-lexicon-german.txt [hunk 2]
```diff
@@ -1,35 +1,3 @@
...
-portal
-profil
-prüfung
-register
-schulung
-variante
-vertrag
-verträge
\ No newline at end of file
+adresse
```
[the remark this answers is not in the document]
Comment (ksuess) [reply examples/analysis/elasticsearch-lexicon-german.txt]: Example configuration with keywords and lexicon, like in tests of @rohberg/volto-searchkit-block

**Comment by jensens** [comment]
Just merge this yourself if you think it is ready.

**Comment by ksuess** [comment]
@jensens Can we get a release please.

**Comment by jensens** [comment]
> @jensens Can we get a release please.

Done.

**Pull request merged**